README.md

```diff
@@ -9,11 +9,13 @@ C API to Qt for Qlift
 - `ENABLE_SHARED`: Enable or disable building qlift-c-api as a shared library. Default is ON.
 - `ENABLE_STATIC`: Enable or disable building qlift-c-api as a static library. Default is OFF.
 
-To install as a shared library in ```/usr/local```:
+To install as a shared library in ```/usr/local``` (default install directory):
 ```sh
 git clone https://github.com/Longhanks/qlift-c-api
 cd qlift-c-api
-cmake CMAKE_INSTALL_PREFIX=/usr/local .
+mkdir build
+cd build
+cmake ..
 make
 sudo make install # root access may not be nessasary if you have /usr/local set up right
```

Comment:
Some more minor caveats:
- "necessary" (typo)
- What do you mean by set up right? `/usr/local` defaults to being owned by root:root and permissions `rwxr-xr-x`. It's unusual to change the owner ([see this](https://unix.stackexchange.com/a/37726)).